policies/linux.consul.-.catalog.nodes_up.json

```diff
@@ -0,0 +1,27 @@
+{
+  "policy": {
+    "actions": [
+      {
+        "category": 2,
+        "type": "event"
+      }
+    ],
+    "conditions": [
+      {
+        "analytic": "baselineDeviation",
+        "metric": "consul.catalog.nodes_up",
+        "operator": "<"
+      }
+    ],
+    "deleted": false,
+    "description": "This policy will generate a WARNING event when the number of active nodes drops.",
```

Comment:
Could you provide a more user friendly description of what this policy is indicating to a user receiving the alert? Any action which should be taken? Given the title of the policy, as a user I'd expect some mention of possible node loss in the description.

Reply:
@Grumblesmith I've commited the requested changes to the branch